server/src/test/java/org/elasticsearch/search/rescore/QueryRescorerBuilderTests.java

```diff
@@ -217,64 +218,58 @@ public void testUnknownFieldsExpection() throws IOException {
                 "    \"window_size\" : 20,\n" +
                 "    \"bad_rescorer_name\" : { }\n" +
                 "}\n";
-        {
-            XContentParser parser = createParser(rescoreElement);
-            Exception e = expectThrows(NamedObjectNotFoundException.class, () -> RescorerBuilder.parseFromXContent(parser));
-            assertEquals("[3:27] unable to parse RescorerBuilder with name [bad_rescorer_name]: parser not found", e.getMessage());
-        }
-
+            try (XContentParser parser = createParser(rescoreElement)) {
```

Comment:
There seems to be an extra level of indentation for all the `try` statements added in this file.